lang/py/src/avro/schema.py

```diff
@@ -315,14 +315,13 @@ def name_ref(self, names):
   namespace = property(lambda self: self.get_prop('namespace'))
   fullname = property(lambda self: self._fullname)
 
+#
 # Logical type class
-class LogicalSchema(Schema):
-  def __init__(self, type, logical_type, other_props=None):
-    self.logical_type = logical_type
+#
 
-    # Call parent ctor
-    Schema.__init__(self, type, other_props)
-    self.set_prop('logicalType', logical_type)
+class LogicalSchema():
```

Comment:
In Python 2, inheriting from `object` will create new style class (https://docs.python.org/release/2.5.2/ref/node33.html). In Python 3, it inherits from the object implicitly.